proto-actor/src/test/kotlin/actor/proto/mailbox/MailboxSchedulingTests.kt

```diff
@@ -3,11 +3,9 @@ package actor.proto.mailbox
 import actor.proto.fixture.TestMailboxHandler
 import actor.proto.fixture.TestMessage
 import actor.proto.fixture.TestSystemMessage
-import org.junit.Test
+import org.junit.jupiter.api.Assertions.*
```

Comment:
About Wildcard imports. I personally don't allow them in our company because sometimes it's hard to determine in the code, where a symbol, function, whatever comes from.
What's your opinion on that?

Reply:
The same

Reply:
I have changed imports to get rid of *